src/libs/entity/entity/EntityManager.cpp

```diff
@@ -1,13 +1,50 @@
 #include "EntityManager.h"
 
 #include "Component.h"
+#include "Archetype.h"
+#include "Blueprint.h"
 
+#include <cassert>
 #include <iterator>
 
 
 namespace ad {
 namespace ent {
 
+Handle<Entity> EntityManager::createFromBlueprint(Handle<Entity> aBlueprint, const char * aName)
```

Comment:
Maybe we can extract a sort of `Entity::copy()` for the API, and use it to implement as a copy then remove.